img/ADBCFlow2.svg

```diff
@@ -0,0 +1 @@
3-38l32 2c-6 37-31 61-74 61-59 0-76-41-82-99-10-93 101-131 147-64 4 7 5 14 7 22l-32 3c-4-21-16-35-41-35" id="ay"/><g id="t"><use transform="matrix(0.07407407407407404,0,0,0.07407407407407404,0,0)" xlink:href="#ax"/><use transform="matrix(0.07407407407407404,0,0,0.07407407407407404,14.814814814814808,0)" xlink:href="#ac"/><use transform="matrix(0.07407407407407404,0,0,0.07407407407407404,23.62962962962962,0)" xlink:href="#ab"/><use transform="matrix(0.07407407407407404,0,0,0.07407407407407404,38.44444444444443,0)" xlink:href="#ag"/><use transform="matrix(0.07407407407407404,0,0,0.07407407407407404,45.85185185185183,0)" xlink:href="#ab"/><use transform="matrix(0.07407407407407404,0,0,0.07407407407407404,60.66666666666664,0)" xlink:href="#ay"/><use transform="matrix(0.07407407407407404,0,0,0.07407407407407404,73.99999999999997,0)" xlink:href="#ab"/><use transform="matrix(0.07407407407407404,0,0,0.07407407407407404,88.81481481481478,0)" xlink:href="#as"/></g><path fill="#3a414a" d="M30-248c118-7 216 8 213 122C240-48 200 0 122 0H30v-248zM63-27c89 8 146-16 146-99s-60-101-146-95v194" id="az"/><g id="u"><use transform="matrix(0.06172839506172835,0,0,0.06172839506172835,0,0)" xlink:href="#az"/><use transform="matrix(0.06172839506172835,0,0,0.06172839506172835,15.987654320987643,0)" xlink:href="#ah"/><use transform="matrix(0.06172839506172835,0,0,0.06172839506172835,28.333333333333314,0)" xlink:href="#ag"/><use transform="matrix(0.06172839506172835,0,0,0.06172839506172835,34.50617283950615,0)" xlink:href="#ah"/></g><path fill="#3a414a" d="M117-194c89-4 53 116 60 194h-32v-121c0-31-8-49-39-48C34-167 62-67 57 0H25l-1-190h30c1 10-1 24 2 32 11-22 29-35 61-36" id="aA"/><path fill="#3a414a" d="M108 0H70L1-190h34L89-25l56-165h34" id="aB"/><path fill="#3a414a" d="M16-82v-28h88v28H16" id="aC"/><g id="v"><use transform="matrix(0.06172839506172835,0,0,0.06172839506172835,0,0)" xlink:href="#ay"/><use transform="matrix(0.06172839506172835,0,0,0.06172839506172835,11.111111111111104,0)" xlink:href="#ab"/><use transform="matrix(0.06172839506172835,0,0,0.06172839506172835,23.456790123456774,0)" xlink:href="#aA"/><use transform="matrix(0.06172839506172835,0,0,0.06172839506172835,35.80246913580244,0)" xlink:href="#aB"/><use transform="matrix(0.06172839506172835,0,0,0.06172839506172835,46.91358024691354,0)" xlink:href="#aw"/><use transform="matrix(0.06172839506172835,0,0,0.06172839506172835,59.25925925925921,0)" xlink:href="#ac"/><use transform="matrix(0.06172839506172835,0,0,0.06172839506172835,66.60493827160488,0)" xlink:href="#aC"/></g><g id="w"><use transform="matrix(0.06172839506172835,0,0,0.06172839506172835,0,0)" xlink:href="#af"/><use transform="matrix(0.06172839506172835,0,0,0.06172839506172835,11.111111111111104,0)" xlink:href="#ai"/><use transform="matrix(0.06172839506172835,0,0,0.06172839506172835,15.987654320987646,0)" xlink:href="#ab"/><use transform="matrix(0.06172839506172835,0,0,0.06172839506172835,28.333333333333314,0)" xlink:href="#aA"/></g></defs></g></svg>
```

Comment:
I think it would be best to add bidirectional arrow heads to the lines connecting **ADBC API** to **Flight SQL ADBC Driver** and **libpq ADBC Driver**

Reply:
Updated the diagram, thanks!